spec/steps/other_info/form_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'rails_helper'
+
+RSpec.describe Steps::OtherInfoForm do
+  let(:form) { described_class.new(application:, **arguments) }
+
+  let(:arguments) do
+    {
+      application:,
+      other_info:,
+      conclusion:,
+      concluded:,
+    }
+  end
+
+  let(:application) { instance_double(Claim, update!: true) }
+  let(:other_info) { 'other relevent information' }
+
+  describe '#save concluded yes' do
```

Comment:
It is fine for now, but should consider that you are testing the exact same things with both the `save` and `valid` tests. Things to keep in mind with this:

* valid are a better cover for these sets of tests as the conditions are related to validity
* save is not a method on the form - it is inherited, as such we really want to use it to test the `persist!` method
* We are not actually testing the persist! method is doing what we want

In your defence this is how I wrote the first couple of sets of tests I did and you have just copied them. Leave these as is for now, and I will try to do a test review at some stage in the future.

Reply:
it did strike me i was testing the same thing twice - should really use common sense - cheers